index.html

```diff
@@ -3,12 +3,57 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/styles.css" />
     <title>Currency converter</title>
 </head>
 <body>
+    <header>
+        <h1>Welcome to New Currency Converter</h1>
+    </header>
     <main>
+        <form>
+            <fieldset class="frame">
+                <legend>Insert rates</legend>
+                <label class="lbl_form" for="base_curr">Base currency</label>
+                <input id="base_curr" type="text" maxlength="3" value="EUR">
```

Comment:
also I think we do not want to change the base right now, so it can be also readonly :)